src/Microsoft.Azure.SignalR.Common/ServiceConnections/MultiEndpointServiceConnectionContainer.cs

```diff
@@ -31,7 +31,7 @@ internal class MultiEndpointServiceConnectionContainer : IMultiEndpointServiceCo
             IServiceEndpointManager endpointManager,
             IMessageRouter router,
             ILoggerFactory loggerFactory,
-            TimeSpan? scaleTimeout = null)
+            TimeSpan? scaleTimeout = null) : base(loggerFactory)
```

Comment:
is it enough to have another ctor for `MultiEndpointServiceConnectionContainer` that supports `endpoints` parameter?

Reply:
No, `MultiEndpointServiceConnectionContainer` will dispose its connection containers while we don't want them to be disposed.

Reply:
@vicancy I prefer inherit multiendpointcontainer in function extension side, and override the getRountedEndpoints method, since the management sdk is already internal visible to function extension side. it minimal the changes in server sdk, and I don't see any meaning to public withEndpoint method, internal only. the new DirectMultiEnpointContainer similar to view in database, instead of a concret table (connection container here)